### DISCUSSION TITLE
Data Table

### DISCUSSION THREAD

**Discussion by bhargavraviya** [nylo-core/nylo#69] [Ideas]
create a custom data table for option given below feature 
- search box
- sort 
- pagination control (custom option) 
- export to csv or excel 
- checkbox index for bulk delete 
- cell edit and delete option

@agordn52

**Comment by agordn52** [comment]
Hi @bhargavraviya,

Thanks for the idea but I think developers will integrate this themselves if their project requires it.

The Nylo's role is to provide a stable, reliable and streamlined approach for creating Flutter apps.
We should think about the common problems developers run into when building apps from scratch.

E.g. handling media, logging data, API requests and deployments. 
I'll make a new discussion and list some areas that I think personally would help developers.
